web/src/components/Toolbar.vue

```diff
@@ -18,20 +18,36 @@
 
     <v-spacer />
 
-    <v-toolbar-items class="d-flex align-center">
-      <v-btn
-        text
-        href="about"
+    <v-toolbar-items
+      v-if="$vuetify.breakpoint.mdAndUp"
+      class="d-flex align-center"
+    >
+      <div
+        v-for="item in filteredItems"
+        :key="item.title"
       >
-        About
-      </v-btn>
+        <v-btn
+          text
+          :href="item.path"
+        >
+          {{ item.title }}
+        </v-btn>
+      </div>
       <v-divider
         class="mx-4"
         inset
         vertical
       />
       <language-switcher />
     </v-toolbar-items>
+    <v-toolbar-items v-else>
+      <v-btn
+        text
+        @click="$emit('click-hamburger')"
```

Comment:
FYI events must be kebab cased per Vue guidelines

Reply:
Btw, should we use the term `menu` instead of `hamburger`? lol

Reply:
sure, it's kinda jargon for the three line menu, but I can change it if you prefer